pallets/common/src/traits/asset.rs

```diff
@@ -64,4 +68,15 @@ pub trait Trait<V, U> {
         ticker: &Ticker,
         value: V,
     ) -> DispatchResultWithPostInfo;
+    fn base_create_asset(
```

Comment:
Could you give this a doc comment? (Use `fn create_asset` as a guide.)

Reply:
Btw, I wouldn't say its a good thing that these trait functions have no documentation.

Reply:
If anything, I'm all for documentation when it adds value, it's just that I'm not a PolyMesh developer and I don't fully understand what the rest of these trait functions are doing, I just didn't want to add a full comment section for this one and leave the rest undocumented.